### DISCUSSION TITLE
Option to disable update-related popups

### DISCUSSION THREAD

**Discussion by mrcjkb** [zen-browser/desktop#4070] [Ideas]
Hey :wave: 

I use zen-browser on NixOS, so the browser's update mechanism doesn't work. The browser gets updated when updating my NixOS configuration.
Whenever there's a new version available, the browser displays a popup, prompting me to download the update.
I haven't found a way to disable this and it's really annoying when it happens, as it keeps popping up again, some time after I dismiss it.

Cheers :smile:

**Comment by leyuuu** [comment]
![image](https://github.com/user-attachments/assets/59c1984f-52e2-4dba-8cf7-d72782e2f05f)

https://github.com/zen-browser/desktop/discussions/896#discussioncomment-11691039
